audio/midi/playback.go

```diff
@@ -0,0 +1,258 @@
+package midi
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"gitlab.com/gomidi/rtmididrv"
+
+	"gitlab.com/gomidi/midi/gm"
+
+	"gitlab.com/gomidi/midi/midimessage/channel"
+
+	"gitlab.com/gomidi/midi/midimessage/meta"
+
+	"gitlab.com/gomidi/midi"
+
+	"gitlab.com/gomidi/midi/mid"
+	"gitlab.com/gomidi/portmididrv"
+)
+
+// FixedMessage wraps a MIDI message and fixes bytes for portmididrv.
+type FixedMessage struct {
+	src midi.Message
+}
+
```

Comment:
can be simplified as `type FixedMessage midi.Message` to omit the need of writing {o}.src.{m} further down the line